single_pages/dashboard/store/orders.php

```diff
@@ -199,7 +199,15 @@
 
                     <form action="<?=URL::to("/dashboard/store/orders/updatestatus",$order->getOrderID())?>" method="post">
                         <div class="form-group">
-                            <?= $form->select("orderStatus",$orderStatuses,$order->getStatus());?>
+                          <select id="orderStatus" name="orderStatus" ccm-passed-value="Pending" class="form-control">
```

Comment:
can't we return an array with translated values from the controller? I think it's bad practice to have this kind of logic in the view.